RecoPixelVertexing/PixelTriplets/plugins/CAHitQuadrupletGenerator.cc

```diff
@@ -131,8 +129,46 @@ namespace {
 				}
 
 			}
+		}
+	}
+  }
+}
+namespace {
```

Comment:
Place all in a single anonymous namespace, i.e. instead of 
```
namespace { <content1> }
namespace { <content2> }
```
do 
```
namespace {
  <content1>
  <content2>
}
```